chains/ethereum/src/utils.ts

```diff
@@ -6,12 +6,13 @@ import type { ModalConnectorsOpts, WalletConnectProviderOpts } from './types'
 
 // -- constants ------------------------------------------------------- //
 export const NAMESPACE = 'eip155'
+const APP_TAG = 'app=2'
```

Comment:
Looks good, no action.

Trying to understand this process. This is just an RPC thing? I read the issue linked but not sure how the RPC identifies between 0 / 1 / 2? Could you explain it a bit more?

Reply:
RPC and Explorer api tag to identify that web3modal is making these requests. 0 and 1 were reserved for other things as per https://github.com/WalletConnect/web3modal/issues/758